seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/dag/physical/PhysicalPlanGenerator.java

```diff
@@ -261,7 +261,7 @@ private List<PhysicalVertex> getEnumeratorTask(List<SourceAction<?, ?, ?>> sourc
         return sources.stream().map(s -> {
             long taskGroupID = idGenerator.getNextId();
             SourceSplitEnumeratorTask<?> t = new SourceSplitEnumeratorTask<>(jobImmutableInformation.getJobId(),
-                new TaskLocation(taskGroupID, convertToTaskID(idGenerator.getNextId(), 1)), s);
+                new TaskLocation(taskGroupID, convertToTaskID(idGenerator.getNextId(), 0)), s);
```

Comment:
Does it need fixing elsewhere?

https://github.com/apache/incubator-seatunnel/blob/st-engine/seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/dag/physical/PhysicalPlanGenerator.java#L231

Reply:
Fixed all calls to `#convertToTaskID`